### PR TITLE
add swaps entry points + swap validation to e2e

### PR DESCRIPTION
**Pull request by BrodyHughes**
Added:
- Entry point from token expanded state
- Entry point from browser screen
- Profile screen / activity validation

### CODE REVIEW

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/brody/swaps-e2e-changes-1/0dcd118498f89e87e2fa722d81a58e6bb357d5c7.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/brody/swaps-e2e-changes-1/0dcd118498f89e87e2fa722d81a58e6bb357d5c7.ipa) for 0dcd118498f89e87e2fa722d81a58e6bb357d5c7

File: src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts [hunk 1]
```diff
@@ -537,7 +538,7 @@ export function useSwapInputsController({
   };
 
   const quoteFetchingInterval = useAnimatedInterval({
-    intervalMs: 12_000,
+    intervalMs: IS_TEST ? 999_999 : 12_000,
```
Comment (brunobar79) [inline src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts:541]: Wouldn't this be a problem and cause flakiness whenever if the quote is no longer valid?

Comment (BrodyHughes) [reply src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts:541]: potentially yeah. i guess the only fix would be to mock the quote then?

Comment (brunobar79) [reply src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts:541]: I can approve after you roll back that change!

**Comment by brunobar79** [comment]
Launch in [simulator](http://localhost:29070/install/ios?virtual=https://app-team.p.rainbow.me/brody/swaps-e2e-changes-1/93a1c3ef915d17c3c81b6eeed18d5b0918edd5c8.app.zip) or [device](http://localhost:29070/install/ios?physical=https://app-team.p.rainbow.me/brody/swaps-e2e-changes-1/93a1c3ef915d17c3c81b6eeed18d5b0918edd5c8.ipa) for 93a1c3ef915d17c3c81b6eeed18d5b0918edd5c8

**Pull request closed without merging**